supporting_content/appendix/quantity_driven_pricing_strategies.md

```diff
@@ -60,7 +60,8 @@ For better comprehension, please refer to the sample price-tiers configuration a
 | GCP       | Flat-rate based pricing                                                            | Not available                        | *TODO: Add sample data* |
```

Comment:
I propose that this PR not get merged until either 1) sample data is added to the `BilledUnitPrice Pattern` column and the TODOs are removed, or 2) the column is struck.  I recognize that that column is not strictly related to this PR -- those TODOs precede the changes -- but as long as the doc is being touched, I feel that that should be addressed.

Reply:
Note this one is already committed. That said, seems fine to remove something we haven't done yet 😜 I'll make a separate suggestion to simplify the applying it...

Reply:
As mentioned in one of the PricingQuantity discussions, this appendix needs a bit more attention. 
I'm unsure if we'll be able to polish everything as planned. That being said, **removing traces of the scoped-out BilledUnitPrice column is a must.**

Reply:
The mentions of BilledUnitPrice have been removed from this appendix via the [ListUnitPrice review PR](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/234), so I'm closing this conversation.

Reply:
Resolving. This is supporting content that we're not planning to bring into the spec till post v1.0.